lib/initializer.rb

```diff
@@ -0,0 +1,16 @@
+class Initializer
+
+  attr_accessor :input, :output, :board
+
+  def initialize(input=STDIN, output=STDOUT)
```

Comment:
What would happen if we ask you to change the UI to be web-based? You'd have to not only change the game setup to use the new UI, but you'd also have to make a change here. Why not just pass in your `Display` class and use its public methods here to retrieve the board size?

Reply:
Yes, I see what you mean. The reason I hesitated passing in the Display class was because I was trying not to create a dependency if I could help it. I've changed the code so that Display is being passed in to Initializer.